core/storage/suggestion/gae_models.py

```diff
@@ -72,6 +74,12 @@
 # The delimiter to be used in score category field.
 SCORE_CATEGORY_DELIMITER = '.'
 
+# Threshold time after which suggestion is considered stale and auto-accepted.
+THRESHOLD_TIME_BEFORE_ACCEPT = 7 * 24 * 60 * 60 * 1000
```

Comment:
Name of constant should include units.

Reply:
Done. I added MSECS. I checked other locations and they had it in the short form too. MILLISECONDS would make the variable name too big I feel :)